build.gradle.kts

```diff
@@ -1,15 +1,31 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+import org.jetbrains.dokka.gradle.DokkaTask
 
 plugins {
     id("org.jetbrains.kotlin.jvm") version "1.3.61" // Apply the Kotlin JVM plugin to add support for Kotlin.
     id("org.jmailen.kotlinter") version "2.3.2" // Lint and formatting for Kotlin using ktlint
     id("com.github.johnrengelman.shadow") version "4.0.1"
+    id("org.jetbrains.dokka") version "0.10.1"
+    id("maven-publish")
+    id("signing")
    
     `java-library` // For API and implementation separation.
 }
 
 val executableName = "fabrikt"
+
 group = "com.cjbooms"
+version = "1.0.0"
+
+val projectUrl = "https://github.com/cjbooms/fabrikt"
+val projectScmUrl = "scm:https://cjbooms@github.com/cjbooms/fabrikt.git"
+val projectScmConUrl = "scm:https://cjbooms@github.com/cjbooms/fabrikt.git"
+val projectScmDevUrl = "scm:git://github.com/cjbooms/fabrikt.git"
+val projectIssueUrl = "https://github.com/cjbooms/fabrikt/issues"
+val projectName = "Fabrikt"
+val projectDesc = "Fabricates Kotlin code from OpenApi3 specifications"
+val projectLicenseName = "Apache License 2.0"
```

Comment:
@cjbooms I put this randomly. I don't know what OSS license you want to release under

Reply:
Yep, that's the one:
https://github.com/cjbooms/fabrikt/blob/master/LICENSE